<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VqmGeneratorDialog</class>
 <widget class="QDialog" name="VqmGeneratorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VQM Generator</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="grpInput">
     <property name="title">
      <string>Input WAV File</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="edtWavPath">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>Select a WAV file...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnBrowseWav">
        <property name="text">
         <string>Browse...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="grpParams">
     <property name="title">
      <string>Parameters</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="lblSampleName">
        <property name="text">
         <string>Sample Name:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="edtSampleName">
        <property name="text">
         <string>growl</string>
        </property>
        <property name="placeholderText">
         <string>e.g. growl, scream, etc.</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblPitch">
        <property name="text">
         <string>Base Pitch (MIDI Note):</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="spnPitch">
        <property name="minimum">
         <double>0.000000000000000</double>
        </property>
        <property name="maximum">
         <double>127.000000000000000</double>
        </property>
        <property name="value">
         <double>60.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lblBegin">
        <property name="text">
         <string>Begin Time (sec):</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="spnBegin">
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="minimum">
         <double>0.000000000000000</double>
        </property>
        <property name="maximum">
         <double>9999.000000000000000</double>
        </property>
        <property name="value">
         <double>0.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lblEnd">
        <property name="text">
         <string>End Time (sec):</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="spnEnd">
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="minimum">
         <double>0.000000000000000</double>
        </property>
        <property name="maximum">
         <double>9999.000000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lblFrameRate">
        <property name="text">
         <string>Frame Rate (fps):</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSpinBox" name="spnFrameRate">
        <property name="minimum">
         <number>50</number>
        </property>
        <property name="maximum">
         <number>500</number>
        </property>
        <property name="value">
         <number>172</number>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="lblHarmonics">
        <property name="text">
         <string>Max Harmonics:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QSpinBox" name="spnHarmonics">
        <property name="minimum">
         <number>8</number>
        </property>
        <property name="maximum">
         <number>128</number>
        </property>
        <property name="value">
         <number>64</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="grpOutput">
     <property name="title">
      <string>Output Directory</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="edtOutputDir">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>Select output directory...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnBrowseOutput">
        <property name="text">
         <string>Browse...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="grpPreview">
     <property name="title">
      <string>Output Preview</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTextEdit" name="txtPreview">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>80</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>VqmGeneratorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>VqmGeneratorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
